### PR TITLE
Stringify all json by any content-type

### PR DESCRIPTION
**Pull request by mwpastore**
* Stringify application/json in additon to application/vnd.api+json
* Inspect the top-level contentType in addition to the Content-Type header

### CODE REVIEW

**Comment by jherdman** [comment]
I'm definitely amenable to this. Would you mind adding some tests though?

File: addon/mixins/ajax-request.js [hunk 1]
```diff
@@ -268,7 +268,13 @@ export default Mixin.create({
     const method = hash.method || hash.type || 'GET';
     const requestData = { method, type: method, url: hash.url };
 
-    if (isJSONAPIContentType(getHeader(hash.headers, 'Content-Type')) && requestData.type !== 'GET') {
+    if (
+      requestData.type !== 'GET'
```
Comment (alexlafroscia) [inline addon/mixins/ajax-request.js]: I don't love this style of breaking the conditional into multiple lines... If you don't want to leave it inline, could you mind making it a function call instead?

Comment (mwpastore) [inline addon/mixins/ajax-request.js]: I should probably break it out into its own npm package. 😉 Sure, I'll inline it.

**Comment by mwpastore** [comment]
@jherdman Will do!

**Comment by mwpastore** [comment]
Okay @alexlafroscia and @jherdman, please take another look and let me know if you'd like any further changes. Thank you!

**Comment by alexlafroscia** [comment]
Ugh, I should really just disable Coveralls. Failing this because of a `0.2%` decrease is ridiculous.

**Comment by alexlafroscia** [comment]
Thanks @mwpastore!

**Comment by mwpastore** [comment]
You're welcome! Thank you!

**Pull request merged**